modules/ROOT/pages/activity-message-tracking.adoc

```diff
@@ -0,0 +1,167 @@
+= Tracking Individual Messages
+
+The message view in Anypoint Partner Manager provides visibility into individual transactions within a transmission. Use the message view to monitor the messages exchanged between partner and suppliers, view the message payload, and see how individual messages are processed and routed.
+
+The message view has one record for each transaction. For example, if an incoming transmission contains 30 transactions, the message view has 30 corresponding records.
+
+A message record contains the following information:
+
+* Business key and other custom attributes that are included in the message type definition
+* Pre-translated and post-translated payloads
+
+Message view records are not created when:
```

Comment:
suggestion to avoid passive voice:
"Partner Manager does not create message view records when:"